tests/reftests/pin.test

```diff
@@ -1068,3 +1068,73 @@ opam-version: "2.0"
 
 No solution found, exiting
 # Return code 20 #
+### :C:e: Make sure opam install works when the opam file changes and the package is pinned
```

Comment:
Depending on the order of merge, these header may need a change in the last rebase (`:C:e:` is also used in another PR)